### PR TITLE
feat(armory-io): added documentation for exposeServices step

### PR DESCRIPTION
**Pull request by Greg-At-Armory**
Resolves Jira: [CDAAS-2156]

Preview pages:
* https://deploy-preview-1899--armory-docs.netlify.app/cd-as-a-service/tasks/deploy/preview-link/
* https://deploy-preview-1899--armory-docs.netlify.app/cd-as-a-service/reference/ref-deployment-file/#strategiesstrategynamecanarystepsexposeservices
* https://deploy-preview-1899--armory-docs.netlify.app/cd-as-a-service/reference/ref-deployment-file/#strategiesstrategynamebluegreenredirecttrafficafterexposeservices

[CDAAS-2156]: https://armory.atlassian.net/browse/CDAAS-2156?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by aimeeu — APPROVED** [review]

File: content/en/cd-as-a-service/reference/ref-deployment-file.md [hunk 1]
```diff
@@ -265,6 +265,14 @@ strategies:
             unit: <seconds|minutes|hours>
         - setWeight:
             weight: <integer>
+        - exposeServices:
+            services:
+              - <service-1>
+              - <service-2>
+              - <service-qn>
```
Comment (danielpeach) [inline content/en/cd-as-a-service/reference/ref-deployment-file.md]: I don't recognize "qn" here. What does that mean? I would understand "n" as "the nth service"

File: content/en/cd-as-a-service/tasks/deploy/preview-link.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+---
+title: Create a Temporary Preview Link to a Deployed Service
+linkTitle: Create Preview Links
+description: >
+  Create a temporary public preview link to a deployed service for testing.
+---
+
+## Overview
+
+ Create a temporary public preview link to a deployed service to verify HTTP services after their deployment. The created link is generated with a random domain prefix and automatically expires after a configured time period.
```
Comment (danielpeach) [inline content/en/cd-as-a-service/tasks/deploy/preview-link.md]: "cryptographically secure" is better than "random"

File: content/en/cd-as-a-service/tasks/deploy/preview-link.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+**Blue/Green strategy**: add an `exposeServices` entry to `strategies.<strategyName>.redirectTrafficAfter` in your [deployment file]({{< ref "cd-as-a-service/reference/ref-deployment-file#strategiesstrategynamebluegreenredirecttrafficafterexposeservices" >}}).
+
+{{< include "cdaas/deploy/preview-link-details.md" >}}
+
+## Find your exposed service URL
+
+### Customer Portal
+
+The **Resources** section of your Deployment details page displays a clickable link to preview your service. CD-as-a-Service  automatically creates the preview link right after the `exposeServices` step completes - no need to refresh the page. On the link itself there is a tooltip with the approximate remaining lifetime of the preview link.
```
Comment (danielpeach) [inline content/en/cd-as-a-service/tasks/deploy/preview-link.md]:
```suggestion
The **Resources** section of your Deployment details page displays a clickable link to preview your service. CD-as-a-Service automatically creates the preview link right after the `exposeServices` step completes - no need to refresh the page. On the link itself there is a tooltip with the approximate remaining lifetime of the preview link. 
```

(two spaces in between "CD-as-a-Service" and "automatically")

File: content/en/includes/cdaas/deploy/preview-link-details.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+      - <service-name-n>
+    ttl:
+      duration: <integer>
+      unit: <seconds|minutes|hours>
+{{< /prism >}}
+
+* `services`: List of one-to-many services to expose
+  * Each service name must match an existing service deployed in the target cluster.
+  * Each service must be in the same namespace as the app you are deploying. 
+  * Each service must define a single HTTP(S) port.
```
Comment (danielpeach) [inline content/en/includes/cdaas/deploy/preview-link-details.md]: sadly just HTTP for now, not HTTPS

File: content/en/includes/cdaas/deploy/preview-link-details.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  * Maximum: 24 hours
+  * Default: 5 minutes
+
+Each link automatically expires after the configured amount of time. The exposed link is not secure.  
+
+You can configure multiple `exposeServices` entries, each having a different expiration time.
+
+**Example** 
+
+In this example, the snippet instructs Armory CD-as-a-Service to create a public URL to the deployed services SVC1 and SVC2. Both automatically expire after 2 hours.
```
Comment (danielpeach) [inline content/en/includes/cdaas/deploy/preview-link-details.md]: I would be consistent in the example naming structure and would back-tick them: `service-name-1` and `service-name-2` (those are the examples given above)

File: content/en/cd-as-a-service/reference/ref-deployment-file.md (hunk 1 above)
Comment (Greg-At-Armory) [reply content/en/cd-as-a-service/reference/ref-deployment-file.md]: type'o

**Pull request merged**